### PR TITLE
 Adding extra packages to fix-spec-file

### PR DESCRIPTION
**Pull request by Yarboa**
Rebase latest changes
Adding howto create PR subpackages

Part of #624

### CODE REVIEW

**Comment by Yarboa** [comment]
contains qm and dropin  
```
repoquery --repo copr:copr.fedorainfracloud.org:packit:containers-qm-657
```

File: .packit.sh [hunk 1]
```diff
@@ -32,3 +32,7 @@ sed -i "s/^Release:.*/Release: ${PACKIT_RPMSPEC_RELEASE}%{?dist}/" ${SPEC_FILE}
 
 # Update Source tarball name in spec
 sed -i "s/^Source0:.*.tar.gz/Source0: %{name}-${HEAD_VERSION}.tar.gz/" ${SPEC_FILE}
+
+# Update build additional rpms in spec
+sed -i 's/\(enable_qm_dropin_img_tempdir \).*/\11/' ${SPEC_FILE}
```
Comment (dougsland) [inline .packit.sh:37]: Question: Is it now auto enabling iot img_tempdir and ROS subpackages in QM now? It's ok from my side (at least in upstream).

Comment (Yarboa) [reply .packit.sh:37]: It was by default. I assume you wanted it in. 
I will add only kvm. once i will solve the failures

File: docs/devel/README.md [hunk 2]
```diff
@@ -258,6 +258,35 @@ Example changing the spec and triggering the build via make (feel free to automa
 $ make clean && VERSION=YOURVERSIONHERE make rpm
 ```
 
+### Let automation create/publish PR rpms
+
+subpuckges could be created by Packit and uploaded
+by Copr to  epo.
```
[the remark this answers is not in the document]
Comment (Yarboa) [reply docs/devel/README.md:264]: sorry to `copr repo`

File: docs/devel/README.md [hunk 3]
```diff
@@ -258,6 +258,35 @@ Example changing the spec and triggering the build via make (feel free to automa
 $ make clean && VERSION=YOURVERSIONHERE make rpm
 ```
 
+### Let automation create/publish PR rpms
+
+subpuckges could be created by Packit and uploaded
```
[the remark this answers is not in the document]
Comment (Yarboa) [reply docs/devel/README.md:263]: Ack

File: docs/devel/README.md [hunk 4]
```diff
@@ -258,6 +261,46 @@ Example changing the spec and triggering the build via make (feel free to automa
 $ make clean && VERSION=YOURVERSIONHERE make rpm
 ```
 
+### Let automation create publish PR subpackages
+
+subpuckges could be created by Packit and uploaded
```
Comment (dougsland) [inline docs/devel/README.md:266]: subpackages. Please open a PR later for that.

**Pull request merged**